pdfmaker/css/_modules/base.scss

```diff
@@ -1111,6 +1111,24 @@ p[class*="ExtractSource"] {
   orphans: 2;
 }
 
+p[class*="Extract-Noteextn"] {
+	text-indent: none;
+	text-align: center;
+	margin: $gridheight;
+	hyphens: none;
+}
+
+span.spanaltfont1span1 {
+  	font-family: FeltpenCom-Medium;
+  	font-size: $sizehwritingprint;
+  	line-height: $gridheight;
```

Comment:
No, I thought it was weird.. but since you included them in the specs in the issue I thought maybe they were required for some reason.  I'll yoink them.  I did test to see if they are even honored in the layout (they are!)